test/common/upstream/health_checker_impl_test.cc

```diff
@@ -2598,6 +2517,18 @@ class TestProdHttpHealthChecker : public ProdHttpHealthCheckerImpl {
 
 class ProdHttpHealthCheckerTest : public HttpHealthCheckerImplTest {
 public:
+  void allocHealthChecker(envoy::config::core::v3::HealthCheck conf) {
```

Comment:
You should't need this, unless it differs from the one in the parent class, in which case I'd comment how it's different.

Reply:
health_checker_ is undeclared and getting some pointer issues.